helm_deploy/templates/service-monitor.yaml

```diff
@@ -1,16 +1,14 @@
-{{- if .Values.serviceMonitor.enabled -}}
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
-  name: application-metrics-monitor
-  namespace: {{ .Values.namespace }}
+  name: {{ include "laa-submit-crime-forms.fullname" . }}-monitor
+  namespace: {{ .Release.Namespace }}
 spec:
   selector:
     matchLabels:
-      app: {{ include "laa-submit-crime-forms.fullname" . }}
+      service: {{ include "laa-submit-crime-forms.fullname" . }}-metrics-svc
   endpoints:
   - port: http
```

Comment:
i don't think we need this. may have been required by prometheus_client?!

Reply:
have removed for now following conversation with ivan and comparing to other servicemonitor config implementation